src/@types/refractor/index.d.ts

```diff
@@ -2,12 +2,15 @@ declare module 'refractor' {
   declare function register(language: object): void;
 
   type RefractorNode = {
-    children: RefractorNode[];
-    tagName: string;
+    children?: RefractorNode[];
+    tagName?: string;
     type: string;
     value: string;
     properties?: {
       className?: string | string[];
+      // A node might have arbitrary properties.
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      [key: string]: any;
```

Comment:
Ooh, I like you can make this a fallback for any keys. I could be wrong, but I don't think Flow let you do this.

Reply:
Yes, and it does not alter the previous keys. I was afraid that `className` type would be overridden, but that's not the case, so that's super nice!